core/modules/wcontrol/Foreman.cc

```diff
@@ -75,15 +77,15 @@ Foreman::Foreman(Scheduler::Ptr                  const& scheduler,
     // It will delete temporary tables that it can identify as being created by a worker.
     // Previous instances of the worker will terminate when they try to use or create temporary tables.
     // Previous instances of the worker should be terminated before a new worker is started.
-    _backend = std::make_shared<wdb::SQLBackend>(_mySqlConfig);
+    _backend = make_shared<wdb::SQLBackend>(_mySqlConfig);
     _chunkResourceMgr = wdb::ChunkResourceMgr::newMgr(_backend);
 
     assert(_scheduler); // Cannot operate without scheduler.
 
     LOGS(_log, LOG_LVL_DEBUG, "poolSize=" << poolSize << " maxPoolThreads=" << maxPoolThreads);
     _pool = util::ThreadPool::newThreadPool(poolSize,  maxPoolThreads, _scheduler);
 
-    _workerCommandQueue = std::make_shared<util::CommandQueue>();
+    _workerCommandQueue = make_shared<util::CommandQueue>();
```

Comment:
So, how come "std::" is explicitly use but here it is not? That seems inconsistent.